indigo_app/views/works.py

```diff
@@ -56,6 +56,49 @@ def determine_place(self):
     def get_context_data(self, **kwargs):
         context = super(WorkViewBase, self).get_context_data(work=self.work, **kwargs)
         context['work_json'] = json.dumps(WorkSerializer(instance=self.work, context={'request': self.request}).data)
+
+        # add other dates to timeline
+        work_timeline = self.work.points_in_time()
+        other_dates = [
+            ('assent_date', self.work.assent_date),
+            ('commencement_date', self.work.commencement_date),
+            ('repealed_date', self.work.repealed_date)
+        ]
+        # add to existing dates (e.g. if publication and commencement dates are the same)
+        for entry in work_timeline:
+            for name, date in other_dates:
+                if entry['date'] == date:
+                    if name == 'commencement_date' and self.work.commencing_work:
```

Comment:
Minor style comment (totally optional):

You're effectively always setting `entry[name] = True`, so you could do that before line 71 and simplify the `if`s. Additionally, it's not going to hurt to have `event['commencing_work']` be None, so you can remove that check too.